WLC/ocr/ocr.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import pickle
+
+from os import environ
+from os.path import dirname, join
+from keras.models import model_from_yaml
+from WLC.utils.singleton import Singleton
+
+# Mute tensorflow debugging information on console
+environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
+
+
+class OCR(metaclass=Singleton):
+
+    def __init__(self):
+        self._load_model()
+        self._load_mapping()
+
+    def _load_model(self):
+        yaml_path = join(dirname(__file__), 'model/model.yaml')
+        h5_path = join(dirname(__file__), 'model/model.h5')
+
+        yaml_file = open(yaml_path, 'r')
+        loaded_model_yaml = yaml_file.read()
+        yaml_file.close()
+
+        model = model_from_yaml(loaded_model_yaml)
+        model.load_weights(h5_path)
+
+        self.model = model
+
+    def _load_mapping(self):
+        mapping_path = join(dirname(__file__), 'model/mapping.p')
+        self.mapping = pickle.load(open(mapping_path, 'rb'))
+
+    def predict(self, char):
+        char = char.reshape(1, 28, 28, 1)
```

Comment:
Does this resize the image? Maybe we should move this into the character class.

Reply:
`reshape(...)` doesn't resize the image but repacks the 2D array into a 1D array because the `keras` model takes that as an input. I think we should leave it in this class as it is tied to the model and not the images.